build.gradle

```diff
@@ -65,8 +66,8 @@ repositories {
 test {
     if (System.env.CI == 'true') {
         testLogging.showStandardStreams = true
-        minHeapSize "1g"
-        maxHeapSize "1g"
+        minHeapSize = "1g"
+        maxHeapSize = "1g"
```

Comment:
`=` not needed.

Reply:
IntelliJ will show this line as an error/warning (Need to find which one).

Reply:
I guess my point here is that it’s not broken, so to reduce the blast radius of any change that’s coming in, please keep it to just the thing you are fixing.